content/platforms/kubernetes/tasks/ldap-on-k8s.md

```diff
@@ -120,7 +120,7 @@ curl -v -k -u "demo@redislabs.com:xxx" -X POST -d @add-user.json -H "Content-Typ
 
 ### Deploying a Redis Enterprise cluster
 
-A Redis Enterprise cluster requires no special setup other to be configured with LDAP as the configuration is outside of the scope of the Redis Enterprise custom resource.
+A Redis Enterprise cluster (REC) requires no special setup other to be configured with LDAP as the configuration is outside of the scope of the Redis Enterprise custom resource.
```

Comment:
```suggestion
A Redis Enterprise cluster (REC) requires no special setup other than to be configured with LDAP since the configuration is outside of the scope of the Redis Enterprise custom resource.
```